mlir/include/mlir/Dialect/LLVMIR/FunctionCallUtils.h

```diff
@@ -50,16 +50,16 @@ LLVM::LLVMFuncOp lookupOrCreatePrintCloseFn(ModuleOp moduleOp);
 LLVM::LLVMFuncOp lookupOrCreatePrintCommaFn(ModuleOp moduleOp);
 LLVM::LLVMFuncOp lookupOrCreatePrintNewlineFn(ModuleOp moduleOp);
 LLVM::LLVMFuncOp lookupOrCreateMallocFn(ModuleOp moduleOp, Type indexType,
-                                        bool opaquePointers);
+                                        bool opaquePointers = true);
```

Comment:
Why not drop this argument entirely?

Reply:
I have a followup that removes all of these in one go, but I cannot stack PRs that use branches of my fork. The functions in `FunctionCallUtils.h` are also used in the LLVM utils helpers from `LLVMCommon`.

Once this and the https://github.com/llvm/llvm-project/pull/71075 have been landed, I'll open the other PR that removes all of this.

Reply:
I just hoped these were truly the last remnants :)

Reply:
These are only the last remnants in the MemRefToLLVM lowering :)